functions/src/backfillToTypesenseFromFirestore.js

```diff
@@ -60,8 +59,7 @@ module.exports = functions.handler.firestore.document
               .import(currentDocumentsBatch);
           functions.logger.info(`Imported ${currentDocumentNumber} documents into Typesense`);
         } catch (error) {
-          functions.logger.error("Import error");
-          console.dir(error.importResults);
+          functions.logger.error("Import error", erro);
```

Comment:
Looks like there's a typo in `erro` here

Reply:
Thank you, fixed in → https://github.com/typesense/firestore-typesense-search/pull/20/commits/d35a145ee1132f6789c37f004374ad6f32b33732